src/components/PopOver/PopOverPortal.js

```diff
@@ -0,0 +1,35 @@
+import { Component } from "react";
+import ReactDOM from "react-dom";
+import PropTypes from "prop-types";
+
+class Portal extends Component {
```

Comment:
This is fine for now, but let’s add a ticket to create a single, reusable portal component